test/common/common.c

```diff
@@ -106,10 +106,10 @@ mp_dump (const char *addr, size_t len) {
 			mp_decode_nil(&addr);
 			break;
 		case (MP_UINT):
-			printf("%zd", mp_decode_uint(&addr));
+			printf("%lld", mp_decode_uint(&addr));
```

Comment:
```
/home/travis/build/tarantool/tarantool-c/test/common/common.c: In function ‘mp_dump’:
/home/travis/build/tarantool/tarantool-c/test/common/common.c:109:4: warning: format ‘%lld’ expects argument of type ‘long long int’, but argument 2 has type ‘uint64_t’ [-Wformat=]
    printf("%lld", mp_decode_uint(&addr));
    ^
/home/travis/build/tarantool/tarantool-c/test/common/common.c:112:4: warning: format ‘%lld’ expects argument of type ‘long long int’, but argument 2 has type ‘int64_t’ [-Wformat=]
    printf("%lld", mp_decode_int(&addr));
    ^
```

Reply:
WBT, why did you change it? I checked on Linux and Mac OS on GCC and Clang and it doesn't give a warning.

Reply:
Now I got it: it was done to match placeholders in our msgpack formatting functions. In this case it should use %llu here (and %lld below) and explicitly cast a decode result to (unsigned long long) and (long long) to avoid the warning. But anyway now I excluded this change from the patchset.